task/init_new.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+/**
+ * @module task/init
+ * Writes a server configuration file.
+ */
+
+var bcrypt = require('bcrypt');
+const path = require('path');
+const read = require('./helper/read-promise');
+const writeFile = require('./helper/write-file-promise');
+const genSalt = require('./helper/gen-salt-promise');
+const numberOfHashIterations = 10;
+const jsonIndent = 2;
+
+const config = {};
+
+console.log('This utility will walk you through creating a config.json file.');
+console.log('It only covers the most common items, and tries to guess sensible defaults.');
+console.log('');
+console.log('general setup for project');
+console.log('');
+
+var myArgs = process.argv.slice(2);
```

Comment:
:art: `const myArgs` or even better

```js
const [, , environment, dashboardHostname, dashboardPort, databaseHostname, databaseName, databaseUsername, databasePassword, databaseDialet] = process.argv;
```

destructure to get the values more directly